### ISSUE TITLE
Add Seed Readiness 

### ISSUE THREAD

**Issue by tedteng** [gardener/dashboard#998]
**What would you like to be added**:

Hi, @jguipi can you elaborate/update the idea as you mention from the DoD log.   correct me if I am wrong you want to add a Seed status for Operator Role in Readiness. 

![image](https://user-images.githubusercontent.com/42234376/115214876-87e1e880-a135-11eb-88a0-4ea10740b43e.png)

**Why is this needed**:
The Readiness display `API`, `control plane`, `node`, and `SC` but without Seed status. 
sometimes operator takes most of the attention to check the issue start from the shoot cluster/CP when the issue raised at the beginning. 
However, the root cause may come from the Seed level (eg quota issue or others. )
Whether we have Readiness for Seed on the `Readiness` will help for people for draft troubleshooting at the beginning.

**Comment by jguipi** [comment]
You described it well @tedteng! It will help DOD pinpoint the root cause of an issue faster and provide good information at the dashboard level. We just need to make sure that this Readiness badge is only visible to DOD to not alarm the customer if they see some red, and it doesn’t affect their cluster.

**Comment by grolu** [comment]
Yes this will be an operator only feature, as we will need to access seed data which should not be visible to regular users.

**Comment by grolu** [comment]
@dkistner @dguendisch @rfranzke
I guess this would be feasible - at least for the `all projects` view, as in this case we have the seed status (in case it is a managed seed). Do you think this is a valuable feature?

**Comment by dguendisch** [comment]
qq: so it would be visible in `all projects` but not in a dedicated shoot view?
It would be better than nothing but is there anything making the dedicated shoot view more complicated to have a seed readiness chip as well?

**Comment by grolu** [comment]
We can also have it on the should details page. We just need to think how we can get the seed status as we are not subscribed to it in this case. The question is if this feature really adds value. If yes, we will proceed and come up with an implementation proposal. But if you consider it as not really useful I would consider closing the ticket as this feature will require some effort so I would not classify it as a "quick win"

**Comment by dguendisch** [comment]
> The question is if this feature really adds value.

Personally I'd see it as a nice to have. It would speed up some few cases by some seconds => nice. But not needed at every price, so if this one is more complicated I think we can also live without it.

**Comment by grolu** [comment]
Thank you for your assessment. I will look into it time-boxed. Depending on the effort we will postpone it.

**Comment by grolu** [comment]
On the shoot detail page we have the information:

<img width="470" alt="Screen Shot 2022-09-13 at 17 15 37" src="https://user-images.githubusercontent.com/35373687/189939996-bca47f3f-a847-48e0-b561-61648b36662a.png">
<img width="1028" alt="Screen Shot 2022-09-13 at 17 16 09" src="https://user-images.githubusercontent.com/35373687/189940088-8b83ddeb-5fdb-441f-871e-c970b925b238.png">

For the cluster list we currently do not have the status of the seed. But I guess we need it there, too?!

**Comment by dguendisch** [comment]
The shoot detail page looks very nice!
For the cluster list it would be nice, again not at every dev effort, but nice to have.

**Comment by grolu** [comment]
Ok. Good to hear that we understood your requirement. However, in the PoC the seed readiness is not reactive and we do not have it on the shoot list. In order to have this, we need to watch the seed resource in the frontend. Doing so, we can also show the seed resources to operators. We first need to do some changes to our backend connection. We will continue with this ticket once we have implemented the foundation.
